internal/pkg/cli/cmd/cmd.go

```diff
@@ -307,36 +307,38 @@ func (root *RootCommand) printError(errRaw error) {
 				root.logger.Infof(`Please use %s.`, errDirNotFound.Expected())
 			}
 			if errDirNotFound.Expected() == dependencies.EmptyDir {
-				modifiedErrs.Append(errors.New("directory is not empty"))
+				modifiedErrs.Append(errors.Wrapf(err, "directory is not empty"))
 			} else {
-				modifiedErrs.Append(errors.Errorf("neither this nor any parent directory is %s", errDirNotFound.Expected()))
+				modifiedErrs.Append(errors.Wrapf(err, "neither this nor any parent directory is %s", errDirNotFound.Expected()))
```

Comment:
Tu chceme zmenit chybovu hlasku, no chybala tu referencia na povodnu chybu.
Na to by sa do `format` muselo pridat `%w` co tu nechceme, tj. nechceme aby bol v novej sprave aj povodny text chyby.

Preto som vytvoril novu funkciu `Wrapf`, ktora zmeni text, ale obsahuje referenciu na povodnu chybu.
Vysledkom som screenshots na zaciatku, tj. vieme kde nastala povodna chyba, aj kde sme zmenili hlasku.